tests/check_broker.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+pid=`ps aux | grep mosquitto | grep -v grep | awk '{print $2}'`
```

Comment:
I'm not sure it's enough to check broker's availability. What if its homonym exists..
How about utilizing `mosquitto_pub` and `mosquitto_sub` to ensure the broker's basic functionality?

Reply:
Nice suggestion! Updated. Please check it again.